index.js

```diff
@@ -13,19 +14,23 @@ var watchify = require('watchify');
 var xtend = require('xtend');
 
 module.exports = function (gulp) {
-	function bundler (target, name, dest) {
+	function doBundle (target, name, dest) {
```

Comment:
While I was figuring out this code,  I changed this.  Just putting it back to what it was, its a good name anyway.